src/ui/services/Background.ts

```diff
@@ -319,3 +276,31 @@ export interface AssetDetail extends IAssetInfo {
   originTransactionId: string;
   issuer?: string;
 }
+
+export enum TransactionType {
+  Issue = 3,
+  Transfer = 4,
+}
```

Comment:
Maybe use and update (if necessary) `waveskeeper-types`?
https://github.com/wavesplatform/waveskeeper-types/blob/master/globals.d.ts#L394 a